### PR TITLE
Add logging for puppeteer server start cmd

### PR DESCRIPTION
**Pull request by kananlanginhooper**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ x ] The commit message follows our guidelines: https://github.com/scullyio/scully/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ x ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Other... Please describe:

## What is the current behavior?

It runs the puppeteer server in the background, without showing the command used to start.

Issue Number: N/A

## What is the new behavior?

It just outputs to the console the command used to start the server.  This is helpful for starting the server manually before build.  Node will see the server is started with the same command, and not start it again.  Instead all requests will be directed to your previously started server, and you can watch the logging or errors that are output from the server.  Otherwise there is no logging of those errors or output from the spawned process.

## Does this PR introduce a breaking change?

- [ ] Yes
- [ x ] No

### CODE REVIEW

File: libs/scully/src/lib/startBackgroundServer.ts [hunk 1]
```diff
@@ -46,11 +49,11 @@ export function startBackgroundServer(scullyConfig: ScullyConfig) {
     }
   ).on('close', (code) => {
     if (+code > 0) {
-      const message = 'Problem starting background servers ' + code;
+      const message = 'Problem starting background server ' + code;
```
Comment (SanderElias) [inline libs/scully/src/lib/startBackgroundServer.ts:52]: There are actually 2 to 3 servers started. not just 1
1. the server that serves the unmodified dist files
2. the server that serves the Scully results, we don't use that, but it makes evaluation of the work done easier
3. optional test data server

File: libs/scully/src/lib/utils/serverstuff/handleUnknownRoute.ts [hunk 2]
```diff
@@ -16,7 +16,7 @@ export const handleUnknownRoute: RequestHandler = async (req, res, next) => {
   if (req.accepts('html')) {
     /** only handle 404 on html requests specially  */
     await loadConfig();
-    const distFolder = join(scullyConfig.homeFolder, scullyConfig.hostFolder || scullyConfig.distFolder);
+    const distFolder = scullyConfig.outDir;
```
Comment (SanderElias) [inline libs/scully/src/lib/utils/serverstuff/handleUnknownRoute.ts:19]: See your other PR

File: docs/Reference/plugins/community-plugins/rss.md [hunk 3]
```diff
@@ -77,12 +65,16 @@ Create a `rss.config.json` in your root with the following properties:
...
   "copyright": "2020 your-domain.com",
   "generator": "Page description",
+  "filename": "feed",
   "feedLinks": {
+    "atom": "https://your-domain.com/feed.atom",
     "json": "https://your-domain.com/feed.json",
-    "atom": "https://your-domain.com/feed.atom"
+    "xml": "https://your-domain.com/feed.xml"
   },
   "outDir": "./dist/static",
```
Comment (SanderElias) [inline docs/Reference/plugins/community-plugins/rss.md:74]: Why don't you use the `outDir` from `scullyConfig`?

**Review by SanderElias — APPROVED** [review]
@jorge, If you are ok with this one, it might be merged.

**Comment by kananlanginhooper** [comment]
@jorgeucano If this is not something you want output by default.  I can change to a CLI option, or log to scully.log.

**Review by jorgeucano — APPROVED** [review]

**Pull request merged**